engine/apps/alerts/paging.py

```diff
@@ -42,7 +42,7 @@ def _trigger_alert(
         deleted_at=None,
         defaults={
             "author": from_user,
-            "verbal_name": "Direct paging",
+            "verbal_name": f"Direct paging ({team.name if team else 'No'} team)",
```

Comment:
Why do we hardcode team name in the name of Integration? What if it changes

Reply:
https://github.com/grafana/oncall/pull/2537#discussion_r1264210758